builders/buildPage.ts

```diff
@@ -3,22 +3,35 @@ import Page from "../models/Page";
 import buildPosts from "../builders/buildPosts";
 
 const buildPage = ($: CheerioSelector) => {
+  const avatar = buildAvatar($);
   const name = buildName($);
   const posts = buildPosts($);
 
-  return new Page({ name, posts });
-}
+  return new Page({ name, posts, avatar });
+};
 
 export default buildPage;
 
 const buildName = ($: CheerioSelector) => {
-  const title = $('title').text();
+  const title = $("title").text();
   let name = title;
 
-  const lastDashIndex = title.lastIndexOf('-');
+  const lastDashIndex = title.lastIndexOf("-");
   if (lastDashIndex !== -1) {
     name = title.substring(0, lastDashIndex - 1);
   }
 
   return name;
-}
+};
+
+const buildAvatar = ($: CheerioSelector) => {
```

Comment:
For now it's fine, but name and avatar should be actually read from post itself. 
Is it actually possible it's posted by someone else on the fb page???

Reply:
I don't think so. Only page admins/editors can add posts on page behalf